### ISSUE TITLE
Provide more flexible network binding options

### ISSUE THREAD

**Issue by dechamps** [agersant/polaris#215]
Currently polaris only supports a `--port` option to change the numerical port to listen on.

It would be nice if the binding could be customized further:

- Making it possible to bind to a specific IP address
- Making it possible to bind to a Unix socket path, which is especially convenient (and more secure) when Polaris is being run behind a reverse proxy on the same machine

**Comment by agersant** [comment]
Good idea. Will definitely add the first one. Probably not the Unix socket paths as that doesn't seem supported by the underlying framework.

**Comment by gunnarrt** [comment]
whats the status on this? 

thanks

**Comment by agersant** [comment]
You can subscribe to this ticket to be notified of updates when there are any.
